docs/github-labels.md

```diff
@@ -0,0 +1,108 @@
+| area/endpoint/identity             | Issues or PRs related to endpoint identity | Any |
+| area/endpoint/selection            | Issues or PRs related to endpoint selection | Any |
+| area/endpoint/type                 | Issues or PRs related to endpoint type | Any |
+| area/ipam                          | Issues or PRs related to IP address management (IPAM) | Any |
+| area/licensing                     | Issues or PRs related to Antrea licensing | Any |
+| area/monitoring/logging            | Issues or PRs related to logging | Any |
+| area/monitoring/auditing           | Issues or PRs related to auditing | Any |
+| area/monitoring/traffic-analysis   | Issues or PRs related to traffic analysis | Any |
+| area/monitoring/mirroring          | Issues or PRs related to mirroring | Any |
+| area/network-policy/action         | Issues or PRs related to network policy actions | Any |
+| area/network-policy/api            | Issues or PRs related to the network policy API | Any |
+| area/network-polciy/failsafe       | Issues or PRs related to network policy failsafes | Any |
+| area/network-policy/lifecycle      | Issues or PRs related to the network policy lifecycle | Any |
+| area/network-policy/match          | Issues or PRs related to matching packets | Any |
+| area/network-policy/named-set      | Issues or PRs releatd to network policy named sets | Any |
+| area/network-policy/precedence     | Issues or PRs related to network policy precedence | Any |
+| area/network-policy/scope          | Issues or PRs related to network policy scope (namespace, global, etc.) | Any |
+| area/network-policy/staging        | Issues or PRs related to staging network policies | Any |
+| area/octant                        | Issues or PRs related to Octant | Any |
+| area/ovs/openflow                  | Issues or PRs related to Open vSwitch Open Flow | Any |
+| area/ovs/ovsdb                     | Issues or PRs related to Open vSwitch database | Any |
+| area/OS/linux                      | Issues or PRs related to the Linux operating system | Any |
+| area/OS/windows                    | Issues or PRs related to the Windows operating system | Any |
+| area/provider/aws                  | Issues or PRs related to aws provider | Any |
+| area/provider/azure                | Issues or PRs related to azure provider | Any |
+| area/provider/gcp                  | Issues or PRs related to gcp provider | Any |
+| area/provider/vmware               | Issues or PRs related to vmware provider | Any |
+| area/routing                       | Issues or PRs related to routing and forwarding | Any |
+| area/security/access-control       | Issues or PRs related to access control | Any |
+| area/security/controlplane         | Issues or PRs related to controlplane security | Any |
+| area/security/dataplane            | Issues or PRs related to dataplane security | Any |
+| area/test                          | Issues or PRs related to tests. | Any |
```

Comment:
Could you make the changes we discusses on Slack? I believe it was:
* add `area/test/infra` if there is an issue with some test infrastructure (Jenkins configuration, Ansible playbook, Kind wrappers, later on Cluster API, ...)
* change the description for `area/test` to indicate that the label is to be used for unit & integration tests

I also suggested reserving `area/test/e2e` for the Antrea-specific e2e tests, and renaming `area/test/conformance` to `area/test/community`, but I didn't hear back from you on that.

Reply:
You're right, I missed those. Updating.